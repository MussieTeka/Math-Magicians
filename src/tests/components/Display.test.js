import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../../components/Display';

describe('Display Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Display value="test value" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
