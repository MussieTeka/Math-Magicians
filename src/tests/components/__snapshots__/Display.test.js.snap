// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component renders correctly 1`] = `
<div
  className="display"
>
  test value
</div>
`;
